### ISSUE TITLE
Tree aria-selected is not narrated 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#15042]
Figure out that JAWS has stopped narrated "aria-selected" state for the multi select tree.
https://fluentsite.z22.web.core.windows.net/components/tree/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#usage-tree-multiselect-example

Bug was reported to the JAWS:
https://github.com/FreedomScientific/VFO-standards-support/issues/432#issuecomment-686385093

JAWS admit it as bug but recommending use "aria-checked".

Did some testing and figure out that "aria-checked" will fix JAWS narration. 
NVDA doesn't narrate not checked state only once user "select" the option and then navigate back it narrate checked.
JAWS with aria-checked narrates both "checked" and "not checked" state. 

So basically implementing "aria-checked" will break a bit NVDA experience, but in overall it will fix more. 

My proposal of the fix is change current "aria-selected" on the treeitem into "aria-checked".

**Comment by paulgildea** [comment]
@kolaps33 Thanks!

@miroslavstastny Bubbling this one up.
